zone/gm_commands/reload.cpp

```diff
@@ -61,7 +61,7 @@ void command_reload(Client *c, const Seperator *sep)
 		return;
 	}
 
-	auto pack = new ServerPacket;
+	ServerPacket* pack;
```

Comment:
Initialize this with `= nullptr` since it might contain a non-null garbage value

Reply:
Done